src/transport.rs

```diff
@@ -33,6 +33,8 @@ pub type MultiplexedPacket = MuxPacket<<BufferPacketPool<SimpleBufferPool> as Pa
 pub type ConnectionChannelsBuilder =
     MessageChannelsBuilder<TaskPoolRuntime, MuxPacketPool<BufferPacketPool<SimpleBufferPool>>>;
 
+pub const HEARTBEAT_PACKET: &[u8] = b"BNT:HEARTBEAT";
```

Comment:
Heartbeat is usually done using an empty packet. There is no need it has any actual content.

Reply:
ok